spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
So without `:defer`, `init` or `:config` is indeed the same thing.

You raised a solid point where even us, we cannot for sure tell the behavior of `use-package`...